services/web/client/source/class/qxapp/desktop/preferences/pages/BasePage.js

```diff
@@ -0,0 +1,47 @@
+/* eslint no-warning-comments: "off" */
+qx.Class.define("qxapp.desktop.preferences.pages.BasePage", {
+  extend: qx.ui.tabview.Page,
+
+  construct: function(title, iconSrc = null) {
+    this.base(arguments, null, iconSrc);
+
+    this.setLayout(new qx.ui.layout.VBox(10).set({
+      spacing: 10,
+      alignX: "center"
+    }));
+
```

Comment:
having the icons above the label might look better in this context

```javascript
this.getChildControl("button").set({
  iconPosition: "top",
  center: true,
  paddingRight: 20
});
```

Reply:
No text